src/devices/ge_coloreffects.c

```diff
@@ -0,0 +1,167 @@
+/* GE Color Effects Remote
+ *
+ * Previous work decoding this device:
+ *    https://lukecyca.com/2013/g35-rf-remote.html
+ *    http://www.deepdarc.com/2010/11/27/hacking-christmas-lights/
+ *
+ * Copyright (C) 2017 Luke Cyca <me@lukecyca.com>, Christian W. Zuckschwerdt <zany@triq.net>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ */
+
+#include "rtl_433.h"
+#include "util.h"
+
+
+// Frame preamble:
+// 11001100 11001100 11001100 11001100 11001100 11111111 00000000
+// c   c    c   c    c   c    c   c    c   c    f   f    0   0
+static const unsigned char preamble_pattern[7] = { 0xcc, 0xcc, 0xcc, 0xcc, 0xcc, 0xff, 0x00};
```

Comment:
use a shorter preamble to make the reception more robust, i.e. `{0xcc, 0xff, 0x00}`